ibis/pandas/execution/window.py

```diff
@@ -66,6 +66,12 @@ def _post_process_group_by_order_by(series, parent, order_by, group_by):
     return series
 
 
+def _win_count(se):
```

Comment:
Do you mind replacing `se` by a descriptive name please? Also, a short docstring would be nice.